client/components/modal/modal.service.js

```diff
@@ -217,8 +217,9 @@ app.factory('Modal', function($rootScope, $uibModal) {
           title: title,
           templateUrl: templateUrl,
           submitFn: function(form, model) {
-            cb(model).$promise.then(function() {
+            cb(model).$promise.then(function(data) {
               confirmDelete.close();
+              console.log(data);
             }, function(err) {
```

Comment:
Oops... this must have been something I left when I was experimenting with modals, will delete after review.